explainer.md

```diff
@@ -587,7 +587,7 @@ dictionary XRSessionCreationOptions {
   attribute EventHandler onfocus;
   attribute EventHandler onend;
 
-  Promise<void> updateRenderState(XRRenderStateInit state);
+  Promise<void> updateRenderState(optional XRRenderStateInit state);
```

Comment:
Sorry if I'm not remembering something... why is this optional?

Reply:
It's a silly WebIDL thing. It has to be optional because none of the members of the dictionary are required. 🤷‍♀️